Scripts/Setup/setupExperiment.sh

```diff
@@ -428,6 +431,10 @@ elif [[ "${DATATYPE}" == 'ERA-I' ]]; then
   ln -sf "${DATADIR}/uv/" 'uv' # pressure level date (3D, 0.7 deg)
   ln -sf "${DATADIR}/sc/" 'sc' # pressure level date (3D, 0.7 deg)
   ln -sf "${DATADIR}/sfc/" 'sfc' # surface data (2D, 0.7 deg)
+elif [[ "${DATATYPE}" == 'ERA5' ]]; then
+  rm -f 'pl' 'sl' 
+  ln -sf "${DATADIR}/pl/" 'pl' # Pressure level date (3D, 0.25 deg).
+  ln -sf "${DATADIR}/sl/" 'sl' # Surface data (2D, 0.25 deg).  
```

Comment:
are we using the 0.25 degree version from NCAR? (I don't remember...)

Reply:
We are using 0.25 degree version from CDS:

https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-pressure-levels?tab=overview
https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels?tab=overview